main.js

```diff
@@ -9,10 +9,10 @@ const Menu = electron.Menu;
 const path = require('path');
 const dataStore = require('./src/data-store');
 
-// Uncomment this to see debugging tools.
-// require('electron-debug')({
-//     showDevTools: true
-// });
+// Show debug tools in debugging mode.
+if (dataStore.debug()) {
+	require('electron-debug')({showDevTools: true});
+}
```

Comment:
yeah, nconf parses command line arguments for you, so I figured we could use that to our advantage. :)